internal/codespaces/connection/connection.go

```diff
@@ -16,10 +16,15 @@ const (
 	clientName = "gh"
 )
 
+type TunnelClient struct {
+	*tunnels.Client
+	connected bool
```

Comment:
Am I missing something, I don't see how this is ever set to `true`?

Reply:
Had some changes that didn't end up getting pushed, should be good now!